README.md

```diff
@@ -42,6 +42,9 @@ Please find full documentation under the `doc/` folder. Key documents:
 
```

Comment:
Move link to the wiki here (between links to in-the-repo docs and reference to MSS site). And refer to the wiki in the following way (since wiki will not be only about building):

Visit our [Github Wiki](https://github.com/Intel-Media-SDK/MediaSDK/wiki) for the detailed setting and building instructions, runtime tips and other information.

Reply:
Done